bigquery/simpleapp/simpleapp.go

```diff
@@ -35,28 +37,41 @@ func main() {
 
 // query returns a slice of the results of a query.
 func query(proj string) (*bigquery.RowIterator, error) {
+	// [START bigquery_simple_app_client]
 	ctx := context.Background()
 
 	client, err := bigquery.NewClient(ctx, proj)
 	if err != nil {
 		return nil, err
 	}
+	// [END bigquery_simple_app_client]
 
 	query := client.Query(
 		`SELECT
-		 APPROX_TOP_COUNT(corpus, 10) as title,
-		 COUNT(*) as unique_words
-		 FROM ` + "`publicdata.samples.shakespeare`;")
+			CONCAT(
+				'https://stackoverflow.com/questions/',
+				CAST(id as STRING)) as url,
+			view_count
+		FROM ` + "`bigquery-public-data.stackoverflow.posts_questions` " +
+		`WHERE tags like '%google-bigquery%'
+		ORDER BY view_count DESC
+		LIMIT 10;`)
 	// Use standard SQL syntax for queries.
 	// See: https://cloud.google.com/bigquery/sql-reference/
 	query.QueryConfig.UseStandardSQL = true
```

Comment:
I think this is default now, can remove?

Reply:
You're right. Done.